alpharaw/sciex.py

```diff
@@ -46,7 +46,7 @@ def _import(self,
         )
         data_dict = wiff_reader.load_sample(self.sample_id,
             centroid = self.centroided,
-            centroid_mz_tol = self.centroid_mz_tol,
+            centroid_ppm = self.centroid_ppm,
```

Comment:
is this change equivalent or do we change behaviour here?

Reply:
Algorithm wise, they are the same; but for the results, they mb a bit different. We are not work much on Sciex data, so it will not affect other repos.